config/samples/dspa_healthcheck.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: datasciencepipelinesapplications.opendatahub.io/v1alpha1
+kind: DataSciencePipelinesApplication
+metadata:
+  name: sample
+spec:
+  apiServer:
+    deploy: true
+    enableSamplePipeline: false
+    cABundle:
```

Comment:
comment this out, with an additional comment that if developing against a cluster using self-signed certs, then uncomment this field